router.py

```diff
@@ -0,0 +1,85 @@
+class Module:
+    __idcounter = 0
+    allmodules = []
+
+    def __init__(self,name):
+        self.name = name
+        self.__class__.allmodules.append(self)
```

Comment:
ist das __class__ überhaupt notwendig?
Es handelt sich dabei ja sowieso um eine statische Klassen variable, welche für alle Instanzen gilt.

```suggestion
        self.allmodules.append(self)
```